app/controllers/placements/organisations/users_controller.rb

```diff
@@ -26,7 +25,7 @@ def set_user
   end
 
   def users
-    @users = @organisation.users
+    @users = policy_scope(@organisation.users)
```

Comment:
I don't think this needs to be memoized:

- `index` already sets `@users` to what it should be
- `show` calls this method and then sets `@user` accordingly

This method is not a `set_users` method, just a helper method to get the scoped `users`.

```suggestion
    policy_scope(@organisation.users)
```